samples/pingpong/pom.xml

```diff
@@ -61,6 +55,11 @@
       <artifactId>awaitility</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Don't understand what this is needed for.

Reply:
Because if you remove it, the tests won't build 😅